src/Maestro/Maestro.Data/Models/DependencyFlow.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Maestro.Data.Models
+{
+    public class DependencyFlowEvent
```

Comment:
Hey @adiaaida I'm curious about this work. Is there a documentation / plan that I can take a look to learn about what's going on?

Reply:
The [PKPI doc](https://github.com/dotnet/arcade/blob/master/Documentation/ProductKPIs.md) is a good place to start on why we are gathering this information (see the dependency flow workflow). The [Roadmap](https://github.com/dotnet/arcade/blob/master/Documentation/Projects/PKPIs/Roadmap.md) also discusses the types of information we are trying to gather with this work.